libs/@guardian/react-crossword/src/components/Controls.tsx

```diff
@@ -424,6 +443,11 @@ export const Controls = () => {
 					}
 					return null;
 				})}
+				<AnagramHelper
+					key={'anagram-helper'}
+					disabled={disableClueControls}
+					tabIndex={-1}
```

Comment:
Is this going to prevent all users from tabbing to the _Anagram Helper_ button? (I night be misunderstanding as some attributes are being set here, and others directly on `ClueButton` further up.)